.gitignore

```diff
@@ -12,3 +12,4 @@ packages/wallet-service/.serverless
 packages/wallet-service/.webpack
 packages/wallet-service/.env*
 .yarn/
+.git
```

Comment:
Question: I'm unsure if this has the expected behavior. The `.git` folder is always ignored by default, so why should we add it here?

If we want to ignore nested `.git` folders, will this command work? Or should we use `**/.git`?

Reply:
This wasn't supposed to be there, fixed!

Thanks